src/exportutils.ts

```diff
@@ -1 +1 @@
-export { generateStarkSigner } from './utils';
+export { generatePrivateKey, createStarkSigner } from './utils';
```

Comment:
Are we happy with the method name generatePrivateKey? Do we want to specify this private key is Stark Private Key?

Reply:
will update change the name to `generateStarkPrivateKey` to be more clear 👍